cloud/services/virtualmachines/client.go

```diff
@@ -84,7 +85,9 @@ func (ac *AzureClient) Delete(ctx context.Context, resourceGroupName, vmName str
 	ctx, span := tele.Tracer().Start(ctx, "virtualmachines.AzureClient.Delete")
 	defer span.End()
 
-	future, err := ac.virtualmachines.Delete(ctx, resourceGroupName, vmName)
+	// TODO: pass variable to force the deletion or not
+	// now we are not forcing.
+	future, err := ac.virtualmachines.Delete(ctx, resourceGroupName, vmName, to.BoolPtr(false))
```

Comment:
for now, should we keep as is? or should we always force?

Reply:
can you file an issue for this please and remove the TODO in code?

Reply:
I think we want to turn it on but we should do some validation, let's do it in a separate PRr

Reply:
opened https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1045

Reply:
The force delete flag causes the VM to not go through the graceful shutdown, so improves the latency for delete.